csrc/scheduler/all_schedulers.h

```diff
@@ -7,7 +7,6 @@
 // clang-format on
 #pragma once
 #include <scheduler/matmul.h>
-#include <scheduler/normalization.h>
```

Comment:
> As individual scheduler classes moves from registry.cpp to their dedicated files, their respective headers will be removed from all_schedulers.h.

Then, what is the purpose of this file? It seems a little strange that the file is named `all_schedulers.h`, but it doesn't give you an access to all the schedulers.

Reply:
Its purpose is to define `enum class TORCH_CUDA_CU_API ScheduleHeuristic`, we can rename it after other individual schedulers are moved from it.

Reply:
Hmm, then we also need to make sure any other code using `all_schedulers.h` has proper includes. It seems there are several like fusion_segmenter.h that rely on `all_schedulers.h`, and I think the header itself is reasonable to have. 

It seems to me it'd be just easier to create a new header like `csrc/scheduler/type.h` and move `ScheduleHeuristic` there. Then `registry.h` would just include `schduler/type.h` rather than `all_schedulers.h`.

Reply:
I was thinking getting rid of `all_schedulers.h` then we can include one-by-one just pick the one we need instead of include all of them. What you suggested also make sense. We also have `all_ops.h`, seems like it is a style in our code base.
I will make this change.
Thanks!